files/tasks/packit-tests.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- name: install and run basic packit tests via ansible playbooks
+  command: ansible-playbook -e "ansible_python_interpreter=/usr/bin/python3" -v -c local -i localhost, files/zuul-tests.yaml
+  args:
+    chdir: "{{ reverse_dir }}"
+- name: install and run packit tests with session recording via ansible playbooks
+  command: ansible-playbook -e "ansible_python_interpreter=/usr/bin/python3" -v -c local -i localhost, files/zuul-tests-session-recording.yaml
+  args:
+    chdir: "{{ reverse_dir }}"
```

Comment:
these are the playbooks coming from packit repo, right? I wonder how we could get rid of the nested `ansible-playbook` execution

Reply:
`these are the playbooks coming from packit repo, right? I` Yes.
` I wonder how we could get rid of the nested ansible-playbook execution` I will try to check more docs, maybe I will find something which can help us here

Reply:
we can do import_tasks do run those tasks dynamically

Reply:
but the files which are called via ansible-playbook command are playbooks, not tasks

Reply:
I have tried  `import_playbook` and `include` options and both seem not to proceed `reverse_dir` variable :(

Reply:
so, can we merge this with ansible-playbook execution for now (as we were talking, it will be probably changed in the future)?